kayenta-web/src/main/java/com/netflix/kayenta/interceptors/MetricsInterceptor.java

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright 2015 Netflix, Inc.
```

Comment:
Left this as I simply copied [this](https://github.com/spinnaker/kork/blob/master/kork-web/src/main/java/com/netflix/spinnaker/kork/web/interceptors/MetricsInterceptor.java) and replaced the path variable lookup with a query param lookup.

Reply:
mention this is copied from the kato url

Reply:
Done.